etc/bash/load_db.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# This file is part of the Php DDD Standard project.
+#
+# Copyright (c) 2017 LIN3S <info@lin3s.com>
+#
+# For the full copyright and license information, please view the LICENSE
+# file that was distributed with this source code.
+#
+# @author Beñat Espiña <benatespina@gmail.com>
+
+$(dirname $0)/../bin/symfony-console doctrine:database:drop --force
```

Comment:
The file name can be misleading, as it deletes without confirmation the database, maybe `drop_and_reload_db.sh`?